e2e/charts/README.md

```diff
@@ -0,0 +1,29 @@
+# Charts
+
+## Nacos
+
+Add `livenessProbe`:
+
+```yaml
+  readinessProbe:
+    httpGet:
+      path: /nacos
+      port: {{ .Values.nacos.serverPort }}
+    failureThreshold: 3
+    initialDelaySeconds: 10
+    periodSeconds: 8
+    successThreshold: 1
+    timeoutSeconds: 1
+```
+
+## Istio Discovery Nacos
```

Comment:
What's the relationship between this config snippet and Istio?

Reply:
This section shows what has been changed in `istio-discovery-nacos` compared to `istio-discovery`.